src/components/queue/QueueSearch.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { SafeAreaView } from 'react-native';
+import { AUTHORIZE_SPOTIFY, HEADERS } from '../../../constants/api/spotify';
+import { HOMESCREEN_BACKGROUND, WHITE } from '../../../constants/colors';
+import { Font } from 'expo';
+import MapScreen from '../map/MapScreen';
+import { addGuest } from '../../../store/actions/userActions';
```

Comment:
is addGuest really being called by the queueSearch screen?

Reply:
nope, fixed in new commit